themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -1,5 +1,14 @@
 <?=$this->doctype()?>
-
+<?php // Set up the search box -- there are three possible cases:
```

Comment:
I would suggest reformatting this as:

```
<?php
  // Set up the search box -- there are three possible cases:
  // 1. No search box was set; we should default to the normal box
  // 2. It was set to false; we should display nothing
  // 3. It is set to a custom string; we should display the provided version
  // Set up default search box if no data was provided from the template;
  // this covers case 1.  Cases 2 and 3 are then covered by logic below.
  if (!isset($this->layout()->searchbox)) {
    $this->layout()->searchbox = $this->context($this)->renderInContext('search/searchbox.phtml', []);
  }
?>
```

...I think that indentation somewhat improves readability.

It might also be worth adding a new comment, to the effect of "We should make sure the search box renders before we build the HTML header, so that any Javascript dependencies can be loaded correctly."

Reply:
Done